src/app/tests/TestCommandInteraction.cpp

```diff
@@ -496,10 +496,15 @@ void TestCommandInteraction::TestCommandHandlerWithWrongState(nlTestSuite * apSu
     NL_TEST_ASSERT(apSuite, err == CHIP_NO_ERROR);
 
     TestExchangeDelegate delegate;
-    commandHandler.mExchangeCtx.Grab(ctx.NewExchangeToAlice(&delegate));
+
+    auto exchange = ctx.NewExchangeToAlice(&delegate, false);
+    commandHandler.mExchangeCtx.Grab(exchange);
+
     err = commandHandler.SendCommandResponse();
 
     NL_TEST_ASSERT(apSuite, err == CHIP_ERROR_INCORRECT_STATE);
+
+    exchange->Close();
```

Comment:
This `Close` call is needed because this is not an initiator exchange so does not start with `WillSendMessage` when it's created?  If so, please document that here so people don't copy the patter in places where it is not appropriate.

Reply:
That's right. Will add comments.